public/docs/_examples/dependency-injection/dart/test/hero_list_component_test.dart

```diff
@@ -0,0 +1,36 @@
+// Simulate a simple test
+
+// Reader should look to the testing chapter for the real thing
+@TestOn('browser')
```

Comment:
browser -> vm

(It's totally cool that this test can run on the command line! It makes the test much faster!)